.vscode/tasks.json

```diff
@@ -4,19 +4,34 @@
     "version": "2.0.0",
     "tasks": [
         {
+            "label": "dev:webview",
             "type": "npm",
-            "script": "watch",
+            "script": "dev:webview",
```

Comment:
As far as I could tell, the `watch` task was no longer being used (since the move to webpack), so I figured it was safe to overwrite this rather than creating another task. @Tatsinnit - let me know if I'm wrong about that.

Reply:
Actually it's still there - I was confused by the diff!